app/services/media_authentication_json.rb

```diff
@@ -0,0 +1,83 @@
+###
+# A class to model media authentication checks
+# as JSON for responding in a JSONP endpoint
```

Comment:
is this really returning JSON or a hash?  doesn't the controller turn it into JSON?